scripts/build/genproto.sh

```diff
@@ -41,8 +41,8 @@ function cleanup {
   rm -rf "${GOPATH}"
 }
 
-cleanup
-trap cleanup EXIT
+#cleanup
```

Comment:
If not used can we remove? GIT anyways keeps the version and usually, this type of comment ends up cluttering and being forgotten.

Reply:
Have not decided yet. Ideally we need to use `go mod` and add to vendors, but `proto` files are omitted in vendors so have to clone entire repo.
will need to improve this script